DominantColor/Shared/DominantColors.swift

```diff
@@ -161,19 +161,19 @@ private func distanceForAccuracy(accuracy: GroupingAccuracy) -> (INVector3, INVe
     case .Low:
         return CIE76SquaredColorDifference
     case .Medium:
-        return CIE94SquaredColorDifference()
+        return { CIE94SquaredColorDifference()(colors: $0) }
```

Comment:
Is this change necessary? I don't recall anything in Swift 1.2 that would make the previous implementation not compile.